_projects/2022/100385586/100385586.Rmd

```diff
@@ -212,6 +207,59 @@ p
 
 ```
 
+### Full code 
+
+The entire process looks like this: 
+
+```{r}
+The_graph <- ggplot(religion) + 
+  aes(GRI, SHI) + 
```

Comment:
I cleaned this up previously. We don't need the full code listed. Please remove this.